.github/workflows/main.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
           - build: windows-x64
             os: windows-latest
             rust: 1.48
-            llvm_url: 'https://github.com/wasmerio/llvm-custom-builds/releases/download/10.x/windows-amd64.tar.gz'
+            # llvm_url: 'https://github.com/wasmerio/llvm-custom-builds/releases/download/10.x/windows-amd64.tar.gz'
```

Comment:
Why not using how own build instead of installing LLVM from `choco`?

Reply:
We don’t have an LLVM build for Apple Silicon

Reply:
This appears to be in the `windows-x64` section, not the `macos-arm64` one